flashcards/backend/decks/models.py

```diff
@@ -1,6 +1,39 @@
 from django.db import models
 
 
+class Flashcard(models.Model):
+    pass
+
+
+class Deck(models.Model):
+    pass
+
+
+class Tag(models.Model):
+    """Model for representing tags for decks
+
+    :param name: Tag name
+    :type name: str
+    :param description: Description for tag
+    :type description: str
+    :param flashcard: Foreign Key to model Flashcard
+    :type flashcard: int
+    :param deck: Foreign Key to model Deck
+    :type deck: int
+
+    """
+
+    name = models.CharField(max_length=24)
+    description = models.TextField()
```

Comment:
risky, i think we should add max_length

Reply:
Already is :)

Reply:
@MWasile are you commenting line 27? because this is TextFiled and it does not have `max_length` option. We can think about a validator, but on default this field does not have it 🙃

Reply:
@martynawitkowska [are you sure? 😛](https://docs.djangoproject.com/en/4.1/ref/models/fields/#textfield)

Reply:
Text from link you provided 😅
"If you specify a max_length attribute, it will be reflected in the [Textarea](https://docs.djangoproject.com/en/4.1/ref/forms/widgets/#django.forms.Textarea) widget of the auto-generated form field. However it is not enforced at the model or database level. Use a [CharField](https://docs.djangoproject.com/en/4.1/ref/models/fields/#django.db.models.CharField) for that."

Reply:
we are not using it for formfield, but in DB it won't make a difference

Reply:
CharField sounds quite okay 😀

Reply:
we can think about validating it in serializer later

Reply:
Summing up deleted comments😅 so I think we can leave TextField here, but add max length for serializer.CharField or to reduce mistakes change the `description` to `CharField` here, because both `CharField` and `TextField` are generating later `serializer.CharField`